lib/migrate/loaders/loaders.js

```diff
@@ -319,13 +319,59 @@ module.exports = function(j, ast) {
 			});
 		});
 
+	/**
+	 *
+	 * Puts options object outside use object into use object
+	 *
+	 * @param {Node} p - object expression ast that has a key for either 'options' or 'use'
+	 * @returns {Node} objectExpression - an use object expression ast containing the options and loader
+	 */
+
+	const fitOptionsToUse = p => {
+		let options;
+		p.value.properties.map(prop => {
```

Comment:
`.map` returns a new array. If you don't assign it, it's better to use `.forEach`.